src/app/create-event/page.tsx

```diff
@@ -0,0 +1,159 @@
+'use client';
+
+import { NextPage } from 'next';
+import nookies from 'nookies';
+import { format } from 'date-fns';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { EventInput } from './EventInput';
+import { EventTextarea } from './EventTextarea';
+import { postEvent } from '../../api/events';
+import { eventSchema } from './schema';
+
+const CreateEventPage: NextPage = () => {
+  const {
+    register,
+    watch,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(eventSchema),
+  });
+
+  const { token } = nookies.get();
+
+  const onSubmit = async (data: any) => {
+    const dateTimeIso = format(new Date(data.date), "yyyy-MM-dd'T'HH:mm:ss'Z'");
+
+    const eventDataFormatted = {
+      ...data,
+      date: dateTimeIso,
+    };
+
+    if (token) {
+      postEvent(eventDataFormatted, token);
+    }
+  };
+
+  const levels = [
+    { value: 'Beginner', label: 'Začátečník' },
+    { value: 'Advanced', label: 'Pokročilý' },
+    { value: 'Expert', label: 'Expert' },
+    { value: 'Any', label: 'Pro každého' },
+  ];
```

Comment:
I will move this to the backend to some references. Feel free to create a ticket for it :)

Reply:
done!

Reply:
Hmm, it is different! But okay, I will need to use IsEventInput for now, and I will also try to consider better implementation.